modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionCard/AccountSubscriptionCard.jsx

```diff
@@ -104,81 +98,55 @@ const AccountSubscriptionCard = ({
 			: DisplayOnCard.Blank;
 	};
 
+	const keysProvisionedContent = displayQuantityOnCard(
+		selectedAccountSubscriptionGroup?.name,
+		accountSubscription?.name
+	);
+
 	return (
 		<ClayCard
 			className={classNames(
-				'border border-light mb-0 mr-4 mt-4 shadow-none',
+				'mb-4 shadow-none d-flex flex-column w-100 cp-account-subscription-card-loading ',
```

Comment:
can you please sort these classNames in alphabetical order

Reply:
Done!!